model/resman.go

```diff
@@ -7,21 +7,34 @@ import (
 	"net/rpc"
 )
 
+import "github.com/kc1212/vgs/common"
+import "github.com/kc1212/vgs/discosrv"
+
 type ResMan struct {
-	Node
-	nodes []Worker
-	jobs  []Job
+	common.Node
+	workers []Worker
+	gsnodes []string
+	jobs    []Job
 }
 
 type ResManArgs struct {
 	Test int // TODO change this an actual job
 }
 
-func RunResMan(n int, port string) {
-	nodes := make([]Worker, n)
+// Run is the main function for ResMan, it starts all its services.
+func Run(n int, id int, addr string, dsAddr string) {
```

Comment:
Ok this one is a bit ambiguous. I said before that we can just call everything `Run`, but that's only if the function is an interface, e.g. `func (gs *GridSdr) Run(...)`. But this one is not the case.